tests/NuGet.Services.AzureSearch.FunctionalTests/Support/TokenizationData.cs

```diff
@@ -48,10 +48,13 @@ public static class TokenizationData
         public static readonly IEnumerable<object[]> LowercasesAndAddsTokensOnCasingAndNonAlphaNumeric = ToMemberData(new Dictionary<string, string[]>
         {
             { "HelloWorld", new[] { "helloworld", "hello", "world" } },
+            { "helloWorld", new[] { "helloworld", "hello", "world" } },
             { "foo2bar", new[] { "foo2bar", "foo", "2", "bar" } },
-            { "HTML", new[] { "html"} },
-            { "HTMLThing", new[] { "htmlthing" } },
-            { "HTMLThingA", new[] { "htmlthinga", "htmlthing", "a" } },
+            { "FOO", new[] { "foo"} },
+            { "FOOBar", new[] { "foo", "bar" } },
+            { "FooBAR", new[] { "foobar", "foo", "bar" } },
+            { "FOOBarBuzz", new[] { "foo", "barbuzz", "bar", "buzz" } },
+            { "FooBARBuzz", new[] { "foobar", "foo", "bar", "buzz" } },
```

Comment:
may be also include set of charachters including split tokens spaces and casings, together? like the highlighted one `FOOBar.Baz Qux`